# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/LIROps.yaml
# Commit: ce799cb566ac
# Full Hash: ce799cb566ac0fb95468118608132be11db6a359
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-24 21:54:20
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r c10609d434ae -r ce799cb566ac js/src/jit/LIROps.yaml
--- a/js/src/jit/LIROps.yaml	Thu Mar 24 17:40:32 2022 +0000
+++ b/js/src/jit/LIROps.yaml	Thu Mar 24 17:56:50 2022 +0000
@@ -2723,9 +2723,18 @@
   arguments:
     offset: size_t
 
+- name: WasmLoadTableElement
+  result_type: WordSized
+  operands:
+    elements: WordSized
+    index: WordSized
+
 - name: WasmDerivedPointer
   gen_boilerplate: false
 
+- name: WasmDerivedIndexPointer
+  gen_boilerplate: false
+
 - name: WasmStoreRef
   operands:
     tls: WordSized